subset/connection/test_macoui

```diff
@@ -17,7 +17,7 @@ java -jar mac_oui/build/libs/mac_oui-all.jar $TARGET_MAC
 
 RESULT_AND_SUMMARY="$(grep "RESULT" $LOCAL_REPORT)"
 grep -v "RESULT" $LOCAL_REPORT >> $REDACTED_LOG
-sleep 5
+sleep 10
```

Comment:
did this do anything?  I"ve been sorting through why that test is flaky -- still don't know yet... but I wouldn't have guessed this would make any difference.